### DISCUSSION TITLE
New Training Data (old Water meter/bad image quality)

### DISCUSSION THREAD

**Discussion by spanzetta** [jomjol/AI-on-the-edge-device#2813] [General]
### Files

I know that the image quality is not perfect at all... but consider that my water meter is very very old and it is in a manhole (under the grass level) ..
I spent a lot of time by trying to find the best settings and camera position/led level.. and this is the best I can obtain
Anyway.. the detected numbers (expecially the main digital ones) are totally wrong
See below some pictures .. 
Is there something to do to improve the detection?
In this way I cannot use it all (it doesn't recognize the numbers at all)
Thanks in advance

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/49818743/b5a10ce0-038c-4f99-b44d-bc681970a131)

![Contatore aperto](https://github.com/jomjol/AI-on-the-edge-device/assets/49818743/e612cdbb-a1b7-431a-acde-f4544ff776b8)

![Report GitHub](https://github.com/jomjol/AI-on-the-edge-device/assets/49818743/0498b2de-8ce4-4bdb-b126-9be016ba6aa8)
![Wrong detection LAST](https://github.com/jomjol/AI-on-the-edge-device/assets/49818743/dc966293-8fc3-430b-bcc6-4676304d45e8)

**Comment by jomjol** [comment]
With this kind of raw images, the detection will get very hard. It does not make sense, to add the label "new training data" here.

**Comment by spanzetta** [comment]
Sorry but I didn't know if I had to report it as bug..

I will try to improve as much as possible the image quality.. but I wonder if this can anyway considered as a bug (I see other reports that exhibit similar problems on detecting numbers)

**Comment by caco3** [comment]
you could train your own model, see the docs.

**Comment by caco3** [comment]
https://jomjol.github.io/AI-on-the-edge-device-docs/Learn-models-with-your-own-images/
